src/evict/README.md

```diff
@@ -0,0 +1,56 @@
+| **`eviction_trigger`**   | When cache usage exceeds this threshold, **application threads**  start assisting in eviction.     |
+| **`eviction_dirty_target`** | The target<sup>1</sup> percentage of **dirty<sup>4</sup>** cache usage that eviction tries to maintain.                      |
+| **`eviction_dirty_trigger`** | When cache usage exceeds this **dirty<sup>4</sup>** threshold, **application threads**  start assisting in eviction.   |
+| **`eviction_updates_target`** | The target<sup>1</sup> percentage of **updates<sup>5</sup>** cache usage that eviction tries to maintain.                 |
+| **`eviction_updates_trigger`** | When cache usage exceeds this **updates<sup>5</sup>** threshold, **application threads** start assisting in eviction.                   |
+
+> **Note:** Target<sup>1</sup> sizes must always be lower than their corresponding trigger<sup>2</sup> sizes.
+
+### Eviction Process
+
+The eviction process involves three components:
+
+- **Eviction Server**: The `eviction server` thread identifies evictable pages/candidates, places them in eviction queues and sorts them based on the **Least Recently Used (LRU)** algorithm. It is a background process that commences when the **target<sup>1</sup> thresholds** are reached.
+- **Eviction Worker Threads**: These threads pop pages from the eviction queues and evicts them. The `threads_max` and `threads_min` configurations in [api_data.py](../../dist/api_data.py) control the maximum and minimum number of eviction worker threads in WiredTiger. They also run in the background to assist the server.
+    > It is possible to run only the eviction server without the eviction worker threads, but this may result in slower eviction as the server thread alone would be responsible for evicting the pages from the eviction queues.
+- **Application Threads Eviction**: 
+    - When eviction threads are unable to maintain cache content, and cache content reaches **trigger<sup>2</sup> thresholds**, application threads begin assisting the eviction worker threads by also evicting pages from the eviction queues.
+    - Another scenario, known as **forced eviction**, occurs when application threads directly evict pages in specific conditions, such as:
+        - Pages exceeding the configured `memory_page_max` size (defined in [api_data.py](../../dist/api_data.py))
+        - Pages with large skip list
+        - Empty Internal pages
+        - Obsolete pages
+        - Pages with long update chains
+        - Pages showing many deleted records
+    > In both cases described above, application threads may experience higher read/write latencies.
+
+### APIs for Eviction
+
+The eviction APIs, defined in `evict.h`, allow other modules in WT to manage eviction processes. Below is a brief description of the functionalities provided by these APIs:
+
+- Interrupting and waking up the eviction server when necessary.
+- Specifying which Btrees to prioritise or exclude from the eviction process.
```

Comment:
We've used British spelling here (prioritise). Can we make sure that `s_string` runs on the new files?

Reply:
Updated in [dae3e51](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/dae3e5108325fa590d353fb2b812fcca3eaacc15). I will create a seperate ticket for allowing s_string to run on .md files

Reply:
[WT-13675](https://jira.mongodb.org/browse/WT-13675) created!